zeebe/engine/src/main/java/io/camunda/zeebe/engine/processing/distribution/CommandDistributionAcknowledgeProcessor.java

```diff
@@ -52,6 +52,8 @@ public void processRecord(final TypedRecord<CommandDistributionRecord> record) {
     final var partitionId = recordValue.getPartitionId();
 
     if (!distributionState.hasPendingDistribution(distributionKey, partitionId)) {
+      distributionState.getMetrics().noPendingDistribution(partitionId, distributionKey);
```

Comment:
this metric has to be proven useful, I have not decided yet 😅

Reply:
It describes the number of "acknowledgements" we get after we already concluded the distribution. This may happen when retries/redistributions are occurring, but I think I would leave this metric out for now - as we could get similar information out of the "number for retries" metric.